waiter/src/waiter/token.clj

```diff
@@ -516,48 +516,61 @@
                            :log-level :warn}))))
 
       nil
-      (do
+      (let [existing-editor (get existing-token-metadata "editor")
+            existing-owner (get existing-token-metadata "owner")
+            owner? (and existing-owner (authz/manage-token? entitlement-manager authenticated-user token existing-token-metadata))
+            editor? (and existing-editor (not owner?)  (authz/run-as? entitlement-manager authenticated-user existing-editor))
+            delegated-user (if editor?
+                             (do ;; allow editor to run with owner privileges
+                               (log/info "promoting editor to owner privileges while editing token"
```

Comment:
Can we update this log message?  Editors don't have the same permissions as owners, so the editor doesn't actually have owner privileges.